storage/SystemInfo/ProcMdstat.cc

```diff
@@ -128,18 +128,20 @@ namespace storage
 	    else
 		d = normalizeDevice(tmp);
 
+	    Device device(d);
```

Comment:
Mark this class as ProcMdstat::Device for clarity to avoid confusion with Storage::Device?